docs/tizen-distribution/how-distribute/item.md

```diff
@@ -0,0 +1,15 @@
+# Item
+
+A pre-registered Item Group will be mapped to the application that is currently being registered (for details on how to register on In App Purchase, refer to the In App Purchase Guide).
+
+## 1) In App Purchase Type & Item Group ID
+
+- You can select an item sales type and specify whether to use it or not. For Commercial Sellers, only applications which meet the conditions below can be registered for Tizen In App Purchase.
+  - Non-Wearable applications of the Linked or Standalone types
+  - Applications in which the application.launch or appmanager.launch permission has been declared.
+
+[![img](media/item1-700x83.png)](media/item1.png)
+
+1. Select ‘Tizen In App Purchase’ as In App Purchase Type.
+2. Click the [Search Item Group ID] button to call a pop-up window. In the Available Item Group pop-up window, select the desired item group and save.[![img](media/item2-650x539.png)](media/item2.png)
```

Comment:
Please add two spaces at the end of this sentence for a line feed between the sentence and the image.